pkg/build/iso.go

```diff
@@ -0,0 +1,65 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+const (
+	xorrisoArgsBase = "-indev %s -outdev %s -map %s /combustion -boot_image any replay -changes_pending yes"
+	xorrisoExec     = "/usr/bin/xorriso"
+	xorrisoLogFile  = "iso-build.log"
+)
+
+func (b *Builder) buildIsoImage() error {
+	cmd, err := b.createXorrisoCommand()
+
+	if err != nil {
+		return err
+	}
+
+	err = cmd.Run()
+	if err != nil {
+		return fmt.Errorf("error running xorriso: %w", err)
+	} else {
+		zap.L().Sugar().Debugf("ISO log file created: %s", b.generateIsoLogFilename())
```

Comment:
I worry it's misleading. If the run was unsuccessful, that file may not exist, further confusing the user.

Reply:
Not sure I understand this. The file creation is separate from the command execution and we do forward `StdErr` to the file as well so it should be populated. Am I missing something?

Reply:
Ah, I see what you're saying. I'm confusing myself; there was a period where I was passing in the log filename to xorriso itself, in which case it may not exist if xorriso shit the bed. I agree it should be removed from the else.